src/version.rs

```diff
@@ -0,0 +1,34 @@
+pub struct VersionInfo {
+    pub cql_stress_version: String,
+    pub cql_stress_build_date: String,
+    pub cql_stress_git_sha: String,
+    pub scylla_driver_version: String,
+    pub scylla_driver_date: String,
+}
+
+mod version_info {
+    include!(concat!(env!("OUT_DIR"), "/version_info.rs"));
+}
+
+pub fn get_version_info() -> VersionInfo {
+    VersionInfo {
+        cql_stress_version: version_info::PKG_VERSION.to_string(),
+        cql_stress_build_date: version_info::COMMIT_DATE.to_string(),
+        cql_stress_git_sha: version_info::GIT_SHA.to_string(),
+        scylla_driver_version: version_info::SCYLLA_VERSION.to_string(),
+        scylla_driver_date: version_info::SCYLLA_RELEASE_DATE.to_string(),
+    }
+}
+
+pub fn format_version_info() -> String {
+    let info = get_version_info();
+    format!(
+        "cql-stress version: {} (commit date: {}, commit sha: {})\n\
```

Comment:
it would be nice if the output format would be a bit easier to parse (json or anything else)

Reply:
I'll make it json, single line

Reply:
fixed